src/storage/src/hummock/iterator/forward_user.rs

```diff
@@ -270,7 +269,7 @@ impl<I: HummockIterator<Direction = Forward>> UserIterator<I> {
                     if self.delete_range_iter.current_epoch() >= epoch {
                         self.stats.skip_delete_key_count += 1;
                     } else {
-                        self.latest_val = Bytes::copy_from_slice(val);
+                        self.latest_val.set(val);
```

Comment:
We do not need to copy it from sub-iterator because we will update  it every times self.iterator.value() update.
So we can just return `self.iterator.value()` as result of `value`